packages/rich-text-links/src/index.ts

```diff
@@ -61,32 +62,37 @@ export function getRichTextEntityLinks(
   };
 
   // const content = (document && document.content) || ([] as Node[]);
-  const addLink = ({ data, nodeType }: Block) => {
+  const addLink = ({ data, nodeType }: Node) => {
     const hasRequestedNodeType = !type || nodeType === type;
 
     if (hasRequestedNodeType && isLinkObject(data)) {
       entityLinks[data.target.sys.linkType].set(data.target.sys.id, data.target.sys);
     }
   };
 
-  visitNodes(document, addLink);
+  if (document) {
```

Comment:
It shouldn't benefit from it as it is a non-exposed function with correct types so we should be able to guarantee that it does not receive `null`. 🙈

I'll move it since it does not matter for this fix but I do think we should assert the types as close to the public facing code as possible

Reply:
> It shouldn't benefit from it as it is a non-exposed function with correct types so we should be able to guarantee that it does not receive null. 🙈

But ... welcome to JS 🙂 I would rather have forward-compatible fix in place that prevents us from running into this again than be super clean on the contract and make everyone consuming the `visitNodes` put in an additional check.